File: front-end/src/app/domain/TeamMatchStat.ts
export class TeamMatchStat {
  constructor(public homeId: number,
              public homeIcon: string,
              public homeName: string,
              public homeScores: number,
              public awayId: number,
              public awayIcon: string,
              public awayName: string,
              public awayScores: number
  ) {
  }
}

File: src/main/java/com/nba/datacenter/domain/TeamMatchStat.java
package com.nba.datacenter.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class TeamMatchStat {
    private int homeId;
    private String homeIcon;
    private String homeName;
    private int homeScores;
    private int awayId;
    private String awayIcon;
    private String awayName;
    private int awayScores;
}
